.devcontainer/bazel-base/Dockerfile

```diff
@@ -100,4 +100,29 @@ RUN ldconfig -v
 RUN ln -s /workspaces/magma/.bazel-cache /var/cache/bazel-cache
 RUN ln -s /workspaces/magma/.bazel-cache-repo /var/cache/bazel-cache-repo
 
+# Convenience aliases to build AGW services
```

Comment:
I thought we would create these as actual bazel aliases using https://bazel.build/reference/be/general#alias ?

Reply:
Yes, we wanted to use bazel aliases and I do not know why I did it this way... I will fix it, sorry!